src/colors.js

```diff
@@ -10,12 +10,30 @@ function getColors(theme) {
 
   switch(theme) {
     case "light":
+
+      // Temp override until Primitives is updated
+      lightColors.success.emphasis = "#2c8747";
+      lightColors.btn.primary.bg = lightColors.success.emphasis;
+      lightColors.btn.primary.hoverBg = lightColors.scale.green[5];
+      lightColors.fg.default = "#1f2328";
+      lightColors.fg.muted = "#656d76";
+
       return lightColors;
     case "light_high_contrast":
       return lightHighContrastColors;
     case "light_colorblind":
         return lightColorblindColors;
     case "dark":
+
+      // Temp override until Primitives is updated
+      darkColors.fg.default = "#e4e8ec";
+      darkColors.fg.muted = "#7a8490";
+      darkColors.accent.fg = "#3f84e4";
```

Comment:
```suggestion
      darkColors.accent.fg = "#2f81f7";
```

I think it has to be this one: https://github.com/primer/primitives/pull/517/files